packages/react/src/components/password-creation-input/password-creation.test.tsx

```diff
@@ -0,0 +1,138 @@
+        const wrapper = mountWithProviders(
+            <PasswordCreationInput
+                validations={[
+                    {
+                        label: 'contains numbers',
+                        isValid: (password) => /\d/.test(password),
+                    },
+                ]}
+                onChange={onChange}
+            />,
+        );
+        const password = 'abc';
+        const event = { target: { name: 'password', value: password } };
+
+        getByTestId(wrapper, 'password-input').simulate('change', event);
+
+        expect(onChange).toHaveBeenCalledWith(expect.objectContaining(event), false);
+    });
+
+    test('calls onChange with password and isValid to true when a custom validation is true', () => {
+        const wrapper = mountWithProviders(
+            <PasswordCreationInput
+                validations={[
+                    {
+                        label: 'contains numbers',
+                        isValid: (password) => /\d/.test(password),
+                    },
+                ]}
+                onChange={onChange}
+            />,
+        );
+        const password = '123';
+        const event = { target: { name: 'password', value: password } };
+
+        getByTestId(wrapper, 'password-input').simulate('change', event);
+
+        expect(onChange).toHaveBeenCalledWith(expect.objectContaining(event), true);
+    });
+
+    test('calls onChange with password and isValid to true when password changes to a valid password', () => {
+        const wrapper = mountWithProviders(
+            <PasswordCreationInput onChange={onChange} />,
+        );
+        const password = 'Pe2345678910';
+        const event = { target: { name: 'password', value: password } };
+
+        getByTestId(wrapper, 'password-input').simulate('change', event);
+
+        expect(onChange).toHaveBeenCalledWith(expect.objectContaining(event), true);
+    });
+
+    test('calls onChange with password and isValid to false when password changes to an invalid password', () => {
+        const wrapper = mountWithProviders(
+            <PasswordCreationInput onChange={onChange} />,
+        );
+        const password = 'Pe23';
+        const event = { target: { name: 'password', value: password } };
+
+        getByTestId(wrapper, 'password-input').simulate('change', event);
+
+        expect(onChange).toHaveBeenCalledWith(expect.objectContaining(event), false);
+    });
+
+    test('sets password type to text when show password button is clicked', () => {
+        const wrapper = mountWithProviders(
+            <PasswordCreationInput onChange={onChange} />,
+        );
+        getByTestId(wrapper, 'show-password-button').simulate('click');
+
+        expect(getByTestId(wrapper, 'password-input').prop('type')).toBe('text');
+    });
+
+    test('sets password strength to weak when there is at least 8 characters', () => {
```

Comment:
Je changerais le nom du test pour `'sets password strength to weak when there is only 8 characters'`